cmd/config_test.go

```diff
@@ -149,16 +183,17 @@ func TestConfigSet(t *testing.T) {
 	var b bytes.Buffer
 	cmdr := DeisCmd{WOut: &b, ConfigFile: cf}
 
-	err = cmdr.ConfigSet("foo", []string{"TRUE=false"})
+	err = cmdr.ConfigSet("foo", []string{"TRUE=false", "SSH_KEY=-----BEGIN OPENSSH PRIVATE KEY-----"})
```

Comment:
This is the integration style test, it just checks that SSH_KEY is mangled and send to the server during config:set. It's only there to ensure the `parseSshKey` code which is covered by the unit tests is called at all, which flavor of encoding (plain/path/base64) we use here doesn't really matter.